checker/checker.go

```diff
@@ -261,6 +264,21 @@ func (c *Checker) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	}
 }
 
+// updateInstruction updates the check result's Instruction
+func (c *Checker) updateInstruction(result *check.Results) {
+	for _, r := range result.Results {
+		if r.State == check.StateSuccess {
+			continue
+		}
+
+		// can't judge by other field, maybe update it later
+		switch r.Extra {
+		case check.AutoIncrementKeyChecking:
```

Comment:
this will also match <https://github.com/pingcap/tidb-tools/blob/adaa9020ebea712134f63808689ed114c2e8f85f/pkg/check/table_structure.go#L366-L370>? and is it suitable?

Reply:
fixed in https://github.com/pingcap/dm/pull/299/commits/ebff68d415214400b5d1af446d5c1b6551ab17fc